functional/FunctionalTest.java

```diff
@@ -1692,6 +1697,13 @@ public static void main(String[] args) {
       System.exit(-1);
     }
 
+    if (System.getenv("MINIO_MINT") != null) {
```

Comment:
We can create files in non mint env @balamurugana and continue the tests. Or skip those tests which require files.